app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveySmileyQuestion.kt

```diff
@@ -0,0 +1,45 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.sp
+import com.kks.nimblesurveyjetpackcompose.ui.theme.Black50
+
+private val SMILEY_EMOJIS = listOf("😡", "😕", "😐", "🙂", "😄")
+
+@Composable
+fun SurveySmileyQuestion() {
+    var selectedIndex by remember { mutableStateOf(-1) }
+    Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.Center) {
+        SMILEY_EMOJIS.forEachIndexed { index, text ->
+            TextButton(onClick = {
+                selectedIndex = index
+            }) {
```

Comment:
I think it's safe to have this on one line since the block is so short, what do you think? 🤔 
```suggestion
            TextButton(onClick = { selectedIndex = index }) {
```